docs/source/working_with_aiida/groups.rst

```diff
@@ -2,22 +2,23 @@ Groups
 ------
 
 Groups provide an additional level of data organization based on some common
```

Comment:
I think we should change these first two sentences into:
Groups are a tool to organize the nodes of the provenance graph into sub sets.
Any number of groups can be created and each group can contain any number of nodes of any type.
Typically, you want to put multiple nodes into a group because they share some common property, and through the group they can easily be referenced.

Reply:
done